spec/rspec/core/formatters/failure_list_formatter_spec.rb

```diff
@@ -7,12 +7,12 @@ module RSpec::Core::Formatters
     it 'produces the expected full output' do
       output = run_example_specs_with_formatter('failures')
       expect(output).to eq(<<-EOS.gsub(/^\s+\|/, ''))
-        |./spec/rspec/core/resources/formatter_specs.rb:4:is marked as pending but passes
-        |./spec/rspec/core/resources/formatter_specs.rb:36:fails
-        |./spec/rspec/core/resources/formatter_specs.rb:40:fails twice
-        |./spec/rspec/core/resources/formatter_specs.rb:47:fails with a backtrace that has no file
-        |./spec/rspec/core/resources/formatter_specs.rb:53:fails with a backtrace containing an erb file
-        |./spec/rspec/core/resources/formatter_specs.rb:71:raises
+        |./spec/rspec/core/resources/formatter_specs.rb:4:Expected example to fail since it is pending, but it passed.
+        |./spec/rspec/core/resources/formatter_specs.rb:36:expected: 2 got: 1 (compared using ==)
+        |./spec/rspec/core/resources/formatter_specs.rb:40:expected: 2 got: 1 (compared using ==)|expected: 4 got: 3 (compared using ==)
```

Comment:
:thinking:, the line number here is the line of the example (`example.location`), that's probably why I finally chose to display the example title because it's what can be found on this line, and saying that the error is from there is confusing.

But presenting the actual failures is indeed better, so maybe we can get the failure actual line number from the exception backtrace and then output two distinct lines for failures like this one?